angular.json

```diff
@@ -364,5 +375,8 @@
       "tags": [],
       "implicitDependencies": ["ui"]
     }
+  },
+  "cli": {
+    "analytics": false
```

Comment:
Why?

Reply:
Apologies, seems VSCode added this automatically.. removed this again.